dev/clint/tests/rules/test_redundant_test_docstring.py

```diff
@@ -0,0 +1,400 @@
+from pathlib import Path
+
+from clint.config import Config
+from clint.linter import Location, lint_file
+from clint.rules.redundant_test_docstring import RedundantTestDocstring
+
+
+def test_no_docstring_in_test_functions(index_path: Path, tmp_path: Path) -> None:
+    tmp_file = tmp_path / "test_something.py"
+    tmp_file.write_text(
```

Comment:
we don't need to save the code in a file. We can just pass code and path to list_file.